create-cert-bundle.py

```diff
@@ -0,0 +1,42 @@
+import os
+import logging
+import sys
+
+logger = logging.getLogger()
+handler = logging.StreamHandler(sys.stdout)
+logger.setLevel(logging.INFO)
+logger.addHandler(handler)
+
+if os.environ.get("HTTPS_PROXY") is None:
+    exit()
+logger.info("Creating certificate bundle with proxy root CA, and installing dependencies")
+try:
+    import certifi
+
+except ImportError:
+    try:
+        from pip._vendor import certifi
+    except Exception as e:
+        logger.exception("Error creating certificate bundle with proxy root CA")
+        exit()
+
+p1 = certifi.where()
+p2 = os.environ.get("GG_ROOT_CA_PATH")
+p3 = './ca-bundle.crt'
+try:
+    with open(p1, 'r') as f1, open(p2, 'r') as f2, open(p3, 'a+') as f3:
```

Comment:
I don't think you should use append mode, just use write so it will simply overwrite the file if it exists.